### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinshaffer**
Updated text to reflect status of EMP500

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,13 @@
...
...P catalogue -- how it is generated and how to use it. The EMP dataset is generated from samples that individual researchers have compiled and contributed to the EMP. Samples from each group of researchers represent individual EMP studies. In addition to analyses by contributing researchers on individual studies, we perform cross-study meta-analyses. EMP 16S Release 1, a meta-analysis of the first 97 16S rRNA amplicon studies, has been published ([article](http://doi.org/10.1038/nature24621), [preprint](https://github.com/biocore/emp/tree/master/papers)), and the code and methods used for that manuscript are provided here. EMP 16S Release 2, currently unpublished, includes additional 16S rRNA amplicon samples, and will likely be included with the EMP multi-omics component (EMP500). We are currently finalizing the EMP500 - a mult-omics meta-analysis of 50 studies including >500 samples each processed for 16S, 18S, ITS amplicon sequencing, shotgun metagenomic sequencing, and metabolic profiling. Methods and standard operating procedures (SOPs) for additional amplicon sequencing, shotgun sequencing, and metabolomics related to EMP 16S release 2 and the EMP500 are provided here as well.
```
Comment (antgonza) [inline README.md:7]: Should this be separated in several paragraphs?

Comment (antgonza) [reply README.md:7]: Also, would it be appropriate to explain that there are 2 releases in the first top paragraph?

Comment (antgonza) [reply README.md:7]: Even perhaps just leaving the top paragraph and then having 2 subsections with each release; this should help to make it clear and in the future just have to add more subsections as more releases are available ... ?

Comment (justinshaffer) [reply README.md:7]: Sure thanks for the input. I'll do that now.

**Pull request merged**